spec.bs

```diff
@@ -2299,7 +2299,7 @@ interface InterestGroupBiddingScriptRunnerGlobalScope
         : InterestGroupScriptRunnerGlobalScope {
   boolean setBid(optional GenerateBidOutput generateBidOutput = {});
   undefined setPriority(double priority);
-  undefined setPrioritySignalsOverride(DOMString key, double priority);
+  undefined setPrioritySignalsOverride(DOMString key, optional double? priority);

```

Comment:
Not 100% sure I got this right for the "can omit, pass null or undefined" behavior.

Reply:
I think if you want to accept all of those and handle all of those, then this looks right.